production-swarm/environment.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+export JOBSERVICE_DB_HOST=192.168.56.10
```

Comment:
There are other external variables which haven't been specified in this file - like `JOB_SERVICE_PORT`.  Also, these new ones are prefixed with `JOBSERVICE_` instead of `JOB_SERVICE_`.